### DISCUSSION TITLE
[bug?] Uncovered properties in mutation testing?

### DISCUSSION THREAD

**Discussion by sinnbeck** [pestphp/pest#1243] [Q&A]
I am trying to get 100% mutation coverage in a filamentphp project. The issue I am having is that it is telling me that a public property isn't covered when mutated. The thing is, I can change it just like the mutation suggest, and suddenly several tests fail.. I feel that this is a bug, as I cannot see a way of fixing it via my own code..

The ListApplications class is in listed in `covers()` in the test file

![image](https://github.com/user-attachments/assets/ebb63ce2-ef3b-4e07-be00-104220a45c23)

**Comment by sinnbeck** [comment]
Ran into another example of this. I am making sure the job has the expected delay

```php
it('is delayed', function () {
    $job = app(ProcessWebhookJob::class);

    expect($job->delay)->toBe(5);

});
```
But it claims that it can change the value, and still work as expected
![image](https://github.com/user-attachments/assets/40b6488b-648e-49dd-9cfa-d0595088df34)

**Comment by troccoli-sgr** [comment]
Same issue here for me too.

In my case a have a Livewire component

```php
class SignIn extends Component
{
    #[Validate('required|email')]
    public string $email = '';

    #[Validate('required|string')]
    public string $password = '';

    // ....
}
```

and a simple Pest test

```php
mutate(SignIn::class);

it('renders successfully', function (): void {
    Livewire::test(SignIn::class)
        ->assertOk()
        ->assertSet('email', '')
        ->assertSet('password', '')
        ->assertViewIs('livewire.sign-in');
});
```

But when I run `XDEBUG_MODE=coverage ./vendor/bin/pest -d memory_limit=1G --compact --mutate` I get

![Screenshot 2025-06-27 at 14 10 54](https://github.com/user-attachments/assets/77eb8e98-9b03-4ff3-99a6-09b04ea46457)
